packages/venia-concept/template.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title><%= htmlWebpackPlugin.options.title %></title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Note to my self: need to check the meta tags in prev templates and try getting them into either here or the app logic.